src/notifiers/pushover.ts

```diff
@@ -0,0 +1,38 @@
+import got from 'got';
+import logger from '../common/logger';
+import { NotifierService } from './notifier-service';
+import { NotificationReason } from '../interfaces/notification-reason';
+import { PushoverConfig } from '../common/config';
+
+export class PushoverNotifier extends NotifierService {
+  private config: PushoverConfig;
+
+  constructor(config: PushoverConfig) {
+    super();
+    this.config = config;
+  }
+
+  async sendNotification(url: string, account: string, reason: NotificationReason): Promise<void> {
+    const L = logger.child({ user: account, reason });
+    L.trace('Sending pushover notification');
+
+    try {
+      await got.post('https://api.pushover.net/1/messages.json', {
+        json: {
+          token: this.config.token,
+          user: this.config.userKey,
+          message: `epicgames-freegames-node needs a captcha solved.`,
```

Comment:
Could you add the `reason` here in the message? The other notifiers have it somewhere.

Reply:
Can do!